src/workerd/io/compatibility-date-test.c++

```diff
@@ -65,7 +66,7 @@ KJ_TEST("compatibility flag parsing") {
       kj::StringPtr expectedOutput,
       kj::ArrayPtr<const kj::StringPtr> expectedErrors = nullptr,
       CompatibilityDateValidation dateValidation = CompatibilityDateValidation::FUTURE_FOR_TEST,
-      bool r2InternalBetaApiSet = false) {
+      bool r2InternalBetaApiSet = false, bool experimental = true) {
```

Comment:
Defaulting `experimental = true` means that you are only testing the experimental flag check in the specific test case that sets this `false` -- since `true` means "don't do a check".

Please change this default to `false`, as this achieves much better test coverage. This way, all the test cases are performing the experimental flag check, verifying it does not have false positives.

Reply:
Done